cts/scheduler/summary/leftover-pending-monitor.summary

```diff
@@ -9,6 +9,7 @@ Current cluster status:
     * Clone Set: promotable-1 [stateful-1] (promotable):
       * Promoted: [ node-3 ]
       * Stopped: [ node-1 node-2 ]
+warning: Support for the Promoted role is deprecated and will be removed in a future release. Use Promoted instead.
```

Comment:
lol this confused me for a minute, cts-cli does s/Master/Promoted/ (which we can drop in main / 3.0 since we longer support --enable-compat-2.0)

Reply:
https://github.com/ClusterLabs/pacemaker/pull/3744